assets/components/survey/contributionsSurveySection.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import PageSection from 'components/pageSection/pageSection';
+import ContributionsSurvey from './contributionsSurvey';
+
+
+// ----- Component ----- //
+
+export default function ContributionsSurveySection() {
+
+  return (
+    <div className="component-contributions-survey-section">
+      <PageSection modifierClass="component-contributions-survey-section">
+        <ContributionsSurvey />
+      </PageSection>
+    </div>
+  );
+
+}
```

Comment:
Could this live within the `ContributionsSurvey` component? If not, and they need to be independent components, would it be possible to split the CSS file into two to reflect this?

Reply:
They need to be separate files sometimes it is it's own section, sometimes it is inserted into a section. 

I can seperate the css.